src/TodoListItem.js

```diff
@@ -1,8 +1,13 @@
 import React from "react";
 
-function TodoListItem({ todo }) {
+function TodoListItem({ todo, onRemoveTodo }) {
+
+    function onClick(){
```

Comment:
Nice use of a handler for your callback function. However, I would reconsider the naming of this handler. It can be a little confusing when you have other `onClick` functions. Remember you want your variable name to portray as much meaning as possible.